database/postgres/dal/news.py

```diff
@@ -27,3 +27,19 @@ def fetch_news(self, order_by=None, limit=None, filters=None):
     def get_news_by_id(self, blob_id: str):
         return self.session.query(News).filter(News.blob_id == blob_id).first()
 
+
+    def get_news_by_real_id(self, news_id: str):
+        return self.session.query(News).filter(News.news_id == news_id).first()
+
+
+    def get_sport_by_name(self, sport_name):
```

Comment:
Does it belong here? Maybe we'd better create SportsDal or smth?

Single responsibility principle. (Unless you meant something else by "sport" here)